### ISSUE TITLE
Error while executing command: invalid credentials

### ISSUE THREAD

**Issue by UnrealShadowz** [tp86/legimi-go#6]
Hi there, I'm not really sure what cause this problem.
Tried also to change login and pw to simple ones but no success.

Any clues? Or could be something wrong with the Kindle Serial Number

=== DEBUG START ===
Session response:
state: 0
error code: 0
in last agreement period: false
is activating: false
server message: ""
max tablets: 0
max smartphones: 0
max einks: 0
max windows: 0
max kindles: 0
max devices: 0
max downloads: 0
downloads left: 0
=== DEBUG END ===

**Comment by tp86** [comment]
Did you register through official Legimi app and/or website and have active subscription? If you're trying to create new account using my script, it definitely will not work. Other than that, you can only make sure your credentials and serial number are ok.

**Comment by UnrealShadowz** [comment]
Yes I did register through the website and have an active subscription. Do I need to preregister the serial number?

**Comment by tp86** [comment]
I don't know exactly if and how you can register Kindle through website. You should have subscription plan that allows Kindle device for sure.

You can always try first accessing your books with official Windows app (could be a bit hard if you don't have access to Windows OS) and only then start using my script.

**Comment by UnrealShadowz** [comment]
Thanks, I will try it now.

**Comment by UnrealShadowz** [comment]
I'm sure that login and pw are correct. I can login on the website, windows app and android. 
Could it be the Kindle serial number but I think this would trigger the "invalid kindle id" error.
Furthermore I can't see that the debug output gives any clue about the error.

**Comment by tp86** [comment]
> Could it be the Kindle serial number but I think this would trigger the "invalid kindle id" error.

Not necessarily. I remember similar situation when I gave wrong serial number and server responded with some Kindle id which was obviously incorrect. Can you double-check? Just remove Kindle id from configuration file or provide some temporary config file path to force asking for serial number.

**Comment by tp86** [comment]
BTW, serial number should be given without spaces even if Kindle displays it as if it had them.

**Comment by 73rrto4r** [comment]
Hello,
Did you try to copy and paste your login and password? It might contain special characters triggering some function in the code for example hyphen or a plus sign when typed in. I believe I came across something like that.

As well - you can synchronise your kindle with an android Legimi app.
